mlir/utils/jenkins/Jenkinsfile.nightly

```diff
@@ -0,0 +1,52 @@
+pipeline {
+    agent {
+        docker {
+            image 'rocm/mlir:rocm4.0-latest'
+            args '--user "$(id -u):$(id -g)" --device=/dev/kfd --device=/dev/dri --group-add video -u 0'
+            label 'rocm'
```

Comment:
Do we have machines that support xdlops in our CI?

Reply:
I don't think so, but I will verify with Omkar.

Reply:
we don't have one in CI cluster. the one we have is behind the firewall.

Reply:
I think we should push to add them in the CI. MI100 has already gone public, I don't see it necessary to be hidden in firewall anymore.

Reply:
@yiqian1 When you clarify with Omkar, could you also ask what it takes to add MI100 node in our CI? Actually could you document this in an independent ticket so we can track the status there.